utils/sv_default_macros.py

```diff
@@ -71,7 +74,7 @@ class DefaultMacros():
     @classmethod
     def ensure_nodetree(cls, operator, context):
         '''
-        if no active nodetree
+        if no active_no nodetree
```

Comment:
??

Reply:
I noticed too. i'll fix in next push. thx.